documentation/rfc/journeys_ridesharing.json

```diff
@@ -0,0 +1,683 @@
+                                    "currency": "EUR"
+                                }
+                            },
+                            "sections": [
+                                {
+                                    "from": {
+                                        "name": "Millau (12100)",
+                                        "id": "admin:fr:12145"
+                                    },
+                                    "to": {
+                                        "name": "Pickup RideSharing",
+                                        "id": "start_ridesharing"
+                                    },
+                                    "departure_date_time": "20180110T115000",
+                                    "arrival_date_time": "20180110T115000",
+                                    "co2_emission": {
+                                        "value": 0,
+                                        "unit": ""
+                                    },
+                                    "geojson": {
+                                        "type": "LineString",
+                                        "properties": [
+                                            {
+                                                "length": 1000
+                                            }
+                                        ],
+                                        "coordinates": [
+                                            [
+                                                3.077759,
+                                                44.10067
+                                            ],
+                                            [
+                                                4.78616,
+                                                43.921963
+                                            ]
+                                        ]
+                                    },
+                                    "duration": 0,
+                                    "type": "crow_fly",
+                                    "id": "section_6_1_0",
+                                    "mode": "walking"
+                                },
+                                {
+                                    "from": {
+                                        "name": "Pickup RideSharing",
+                                        "id": "start_ridesharing"
+                                    },
+                                    "to": {
+                                        "name": "Dropoff RideSharing",
+                                        "id": "end_ridesharing"
+                                    },
+                                    "departure_date_time": "20180110T115000",
+                                    "arrival_date_time": "20180110T151000",
+                                    "additional_informations": [
+                                        "date_time_estimated"
```

Comment:
adding this little warning (consistent with what we have in departures but never gonna be used by anyone, I guess)